modules/platforms/cpp/client/include/ignite/table/table.h

```diff
@@ -38,8 +38,8 @@ class Table {
 
 public:
     // Deleted
-    Table(const Table &) = delete;
-    Table &operator=(const Table &) = delete;
+    Table(const Table &) = default;
+    Table &operator=(const Table &) = default;
```

Comment:
Now the comment above is false.

Reply:
No, they should actually still be deleted :) Fixed.